tests/test-pull-repeated.sh

```diff
@@ -23,9 +23,10 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
-echo "1..2"
+echo "1..4"
 
 COMMIT_SIGN="--gpg-homedir=${TEST_GPG_KEYHOME} --gpg-sign=${TEST_GPG_KEYID_1}"
+BIG_NUMBER_RETRIES=8 # Gives error rate <0.5%, with --random-408s=50
```

Comment:
Minor: we don't have to make it a constant upfront. Let's just inline it where it's used? (Though the comment is definitely useful! Let's move that down there too!)

Reply:
Makes sense, making this edit now. Thanks!
One commit plus a prep commit sounds good! Thank you for the link on rebasing, looking over this.